openssl-dynamic/src/main/c/bb.c

```diff
@@ -30,6 +30,7 @@
  */
 
 #include "tcn.h"
+#include "bb.h"
 
 TCN_IMPLEMENT_CALL(jlong, Buffer, address)(TCN_STDARGS, jobject bb)
```

Comment:
`TCN_IMPLEMENT_CALL` and `TCN_METHOD_TABLE_ENTRY` should no longer exist nor be used in `netty-tcnative`. All of these methods should be static. We want to be sure the methods can only be registered with JNI via the manual registration, and we never want the implicit name based registration to occur.